bokehjs/src/coffee/models/annotations/label_set.coffee

```diff
@@ -22,13 +22,25 @@ export class LabelSetView extends TextAnnotationView
       @connect(@model.change, () ->
         @set_data(@model.source)
         @render())
+      @connect(@model.source.streaming, () ->
+        @set_data(@model.source)
+        @render())
+      @connect(@model.source.patching, () ->
+        @set_data(@model.source)
+        @render())
       @connect(@model.source.change, () ->
         @set_data(@model.source)
         @render())
     else
       @connect(@model.change, () ->
         @set_data(@model.source)
         @plot_view.request_render())
+      @connect(@model.source.streaming, () ->
+        @set_data(@model.source)
+        @plot_view.request_render())
+      @connect(@model.source.patching, () ->
+        @set_data(@model.source)
+        @plot_view.request_render())
```

Comment:
@mattpap is it possible to connect multiple signals to the same callback with one call to `@connect`